README.md

```diff
@@ -1,84 +1,107 @@
+
+## Introduction
+
+The Home Mortgage Disclosure Act (HMDA) requires many financial institutions to maintain, report, and publicly disclose information about mortgages. HMDA was originally enacted by Congress in 1975 and is implemented by [Regulation C](https://www.gpo.gov/fdsys/pkg/CFR-2012-title12-vol8/xml/CFR-2012-title12-vol8-part1003.xml). The Dodd-Frank Act transferred HMDA rulemaking authority from the Federal Reserve Board to the Consumer Financial Protection Bureau (CFPB) on July 21, 2011.
+
+This regulation provides the public loan data that can be used to assist:
+* in determining whether financial institutions are serving the housing needs of their communities;
+* public officials in distributing public-sector investments so as to attract private investment to areas where it is needed;
+* and in identifying possible discriminatory lending patterns.
+
+This regulation applies to certain financial institutions, including banks, savings associations, credit unions, and other mortgage lending institutions.
+
+## The HMDA Platform
 
-# Project Title
+This repository contains the code for the entirety of the HMDA platform backend. This platform has been designed to accomodate the needs of the HMDA filing process by financial institutions, as well as the data management and publication needs of the HMDA data asset. 
 
-**Description**:  Put a meaningful, short, plain-language description of what
-this project is trying to accomplish and why it matters.
-Describe the problem(s) this project solves.
-Describe how this software can improve the lives of its audience.
+The HMDA Platform is composed of the following modules: 
 
-Other things to include:
+### Parser
 
-  - **Technology stack**: Indicate the technological nature of the software, including primary programming language(s) and whether the software is intended as standalone or as a module in a framework or other ecosystem.
-  - **Status**:  Alpha, Beta, 1.1, etc. It's OK to write a sentence, too. The goal is to let interested people know where this project is at. This is also a good place to link to the [CHANGELOG](CHANGELOG.md).
-  - **Links to production or demo instances**
-  - Describe what sets this apart from related-projects. Linking to another doc or page is OK if this can't be expressed in a sentence or two.
+Component responsible for reading incoming data and making sure that it conforms to the HMDA File Specification
 
+### Data Validation
 
-**Screenshot**: If the software has visual components, place a screenshot after the description; e.g.,
+Module responsible for validating incoming data by executing validation rules as per the Edit Checks documentation
 
-![](https://raw.githubusercontent.com/cfpb/open-source-project-template/master/screenshot.png)
+### Persistence
+
+This module is responsible for persisting information into the system. It becomes the system of record for HMDA data
```

Comment:
_~~This~~ module ~~is~~ responsible for..._ for consistency with above sections?